src/modeler/AutoRest.Swagger/SwaggerParser.cs

```diff
@@ -22,14 +23,22 @@ public static ServiceDefinition Load(string path, IFileSystem fileSystem)
                 throw new ArgumentNullException("fileSystem");
             }
 
-            return SwaggerParser.Parse(fileSystem.ReadFileAsText(path));
+            var swaggerDocument = fileSystem.ReadFileAsText(path);
```

Comment:
I'm ok with this for now, but I think long-term we're going to want to get to the point where we can hand the `Parser` a DOM of some sort, and isolate it from even knowing about 'files'